app/controllers/home_controller.rb

```diff
@@ -0,0 +1,8 @@
+class HomeController < ApplicationController
+
+  def index
+    @movies = Movie.all.sort_by { |movie| -movie.rank }.first(10)
+    @albums = Album.all.sort_by { |album| -album.rank }.first(10)
+    @books = Book.all.sort_by { |book| -book.rank }.first(10)
```

Comment:
If this application had to be maintained (and expanded to other media types) in the future, this would be a great case for using a constant to store the `10`.